app/js/services/situationService.js

```diff
@@ -120,6 +120,7 @@ angular.module('ddsCommon').factory('SituationService', function($http, $session
             }
 
             var result = {
+                referenceId: situation._id,
```

Comment:
I don't find this name very clear. Pretty much anything could be a “reference”. What about `parent`, `parentSituation`, `previousVersion`, `modifiedFrom`… ?